server/src/main/java/org/elasticsearch/index/engine/EngineConfig.java

```diff
@@ -81,6 +82,7 @@
     private final CircuitBreakerService circuitBreakerService;
     private final LongSupplier globalCheckpointSupplier;
     private final Supplier<RetentionLeases> retentionLeasesSupplier;
+    private final MapperService mapperService;
```

Comment:
I am not entirely sure we need the mapper service, can you double check if we do, I think we can load only the ID and be fine?

Reply:
I removed the mapper service.